jpf-symbc/src/main/gov/nasa/jpf/symbc/SymbolicListener.java

```diff
@@ -341,12 +355,9 @@ public void instructionExecuted(VM vm, ThreadInfo currentThread, Instruction nex
                     Object symbolicVar = sf.getOperandAttr();
 
                     if(interceptSymbolic && strInsn.contains("nativereturn") && strInsn.contains("makeSymbolic")){
-                        symVarName = symbolicVar.toString();
-                        SymbolicVariableInfo Info = new SymbolicVariableInfo(symLineNumber, symRetrunType, symVarName);
-                        symbolicVariableInfoList.add(Info);
-                        symLineNumber = 0;
-                        symRetrunType = "";
-                        symVarName = "";
+                        symbolicVariableInfo.varName = symbolicVar.toString();
+                        symbolicVariableInfoList.add(symbolicVariableInfo);
+
                         // Initialize interceptSymbolic
                         interceptSymbolic = false;
```

Comment:
can you have a one line comment on the definition of this variable that describes why you have it

Reply:
@sohah , I added comments for this variable. Thank you!